public/js/client.js

```diff
@@ -16,12 +18,12 @@ var pos = {x: 0, y: 0, cmd: null};
  * {integer} pos.y mouse y offset
  * {string} pos.cmd key command or mouse click (not implemented yet)
  */
-var emitMouse = function(x, y, cmd) {
-  pos.x = x;
-  pos.y = y;
-  pos.cmd = cmd;
+var emitMouse = function (x, y, cmd) {
+    pos.x = x;
```

Comment:
You can run `gulp lint` to check if all styles are settled.

hint: keep `2 spaces` as indent, and don't put space before brace arguments.